core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/CopyIndexCommand.java

```diff
@@ -71,21 +78,46 @@ public boolean prepare(
 
 	@Override
 	public void execute(
-			OperationParams params ) {
+			final OperationParams params ) {
+		copyIndex(params);
+
+	}
+
+	public Void computeResults(
+			final OperationParams params ) {
+
+		try {
+			copyIndex(params);
+		}
+		catch (WritePropertiesException | ParameterException e) {
+			// TODO GEOWAVE-rest-project server error status message
```

Comment:
same

Reply:
CopyIndex command isn't a service enabled command so I can't set status. The copy commands don't have any endpoints atm, so I think it makes sense to pass here